src/main/java/lotto/service/LottoGame.java

```diff
@@ -0,0 +1,117 @@
+package lotto.service;
+
+import lotto.data.Lotto;
+import lotto.data.LottoNumber;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static lotto.data.LottoNumber.*;
+
+public class LottoGame {
+    private final static int LOTTO_PRICE = 1000;
+    private static final int MIN_LOTTO_NUMBER = 1;
+    private static final int MAX_LOTTO_NUMBER = 45;
+    private static final int WINNING_PRICE_ZERO = 0;
+
+    private LottoGame(){
+        //
+    }
+
+    public static List<Lotto> getLottoList(int amount) {
+        List<Lotto> lottoList = new ArrayList<>();
+        for (int i = 0; i < amount; i++) {
+            lottoList.add(new Lotto());
+        }
+
+        return lottoList;
+    }
+
+    public static int getAmountOfLotto(int price) {
+        return price / LOTTO_PRICE;
+    }
+
+    public static List<Integer> makeLottoNumberPool() {
+        List<Integer> lottoNumberPool = new ArrayList<>();
+        for (int i = MIN_LOTTO_NUMBER; i <= MAX_LOTTO_NUMBER; i++) {
+            lottoNumberPool.add(i);
+        }
+        return lottoNumberPool;
+    }
+
+    public static int matchWinningNumbers(List<Integer> winningNumbers, Lotto lotto) {
+        int matched = 0;
+        for (Integer winningNumber : winningNumbers) {
+            matched = getMatched(lotto, matched, winningNumber);
+        }
+
+        return matched;
+    }
+
+    private static int getMatched(Lotto lotto, int matched, Integer winningNumber) {
+        if (lotto.getNumbers().contains(winningNumber)) {
+            matched++;
+        }
+        return matched;
+    }
+
+    public static void checkWinningNumbers(List<Integer> winningNumbers) {
+        if(winningNumbers.size() != 6){
+            throw new IllegalArgumentException("숫자 6개를 입력해주세요.");
+        }
+
+        List<Integer> distinctList = winningNumbers.stream().distinct().collect(Collectors.toList());
```

Comment:
```suggestion
        List<Integer> distinctList = winningNumbers.stream()
        .distinct()
        .collect(Collectors.toList());
```

가독성과 디버깅을 위해서 저는 이렇게 합니다.